spec/system/show_work_spec.rb

```diff
@@ -4,7 +4,8 @@
 
 RSpec.describe 'Show a work', :rack_test do
   let(:druid) { druid_fixture }
-  let!(:work) { create(:work, druid: druid, title: title_fixture) }
+  let!(:collection) { create(:collection) }
+  let!(:work) { create(:work, druid: druid, title: title_fixture, collection:) }
```

Comment:
Are the `let!` necessary over a normal `let` here? Not a blocker, mostly curious if you found this was required.

Reply:
Hmm, I'm not sure that it is, although the `let!` was there in the code before for instantiating `work`, so perhaps we were anticipating a situation where it would be helpful to not create lazily? I removed it for now.